deploy/twingate-operator/tests/__snapshot__/pre-delete-cleanup_test.yaml.snap

```diff
@@ -1,5 +1,158 @@
-should enable pre-delete cleanup job:
+should enable pre-delete cleanup job when Kubernetes Access Gateway is enabled:
   1: |
+    apiVersion: rbac.authorization.k8s.io/v1
+    kind: ClusterRole
+    metadata:
+      name: RELEASE-NAME-kubernetes-access-gateway-cluster-role
+    rules:
+      - apiGroups:
+          - ""
+        resources:
+          - users
+          - groups
+          - serviceaccounts
+        verbs:
+          - impersonate
+  2: |
+    apiVersion: rbac.authorization.k8s.io/v1
+    kind: ClusterRoleBinding
+    metadata:
+      name: RELEASE-NAME-kubernetes-access-gateway-cluster-role-binding
+    roleRef:
+      apiGroup: rbac.authorization.k8s.io
+      kind: ClusterRole
+      name: RELEASE-NAME-kubernetes-access-gateway-cluster-role
+    subjects:
+      - kind: ServiceAccount
+        name: RELEASE-NAME-kubernetes-access-gateway
+        namespace: NAMESPACE
+  3: |
+    apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      labels:
+        app.kubernetes.io/instance: RELEASE-NAME
+        app.kubernetes.io/managed-by: Helm
+        app.kubernetes.io/name: kubernetes-access-gateway
+        app.kubernetes.io/version: 0.7.0
+        helm.sh/chart: kubernetes-access-gateway-0.7.0
+      name: RELEASE-NAME-kubernetes-access-gateway
+    spec:
+      replicas: 1
+      selector:
+        matchLabels:
+          app.kubernetes.io/instance: RELEASE-NAME
+          app.kubernetes.io/name: kubernetes-access-gateway
+      template:
+        metadata:
+          annotations:
+            checksum/tlsAlternativeNames: eee78e4712e57977935ceb9ebfdb354aa33c85ca4dbb490001827f64259b6b8d
+          labels:
+            app.kubernetes.io/instance: RELEASE-NAME
+            app.kubernetes.io/managed-by: Helm
+            app.kubernetes.io/name: kubernetes-access-gateway
+            app.kubernetes.io/version: 0.7.0
+            helm.sh/chart: kubernetes-access-gateway-0.7.0
```

Comment:
This snapshot test would fail in dependabot PR updating the Gateway subchart (e.g. https://github.com/Twingate/kubernetes-operator/pull/725). Is there a way to avoid having to manually fix those dependabot PR?

Reply:
Yes, it was already fixed in this [commit](https://github.com/Twingate/kubernetes-operator/pull/728/commits/1f8bfacee713e95c3e2ea001fbe3f4ecfd89c7b6)

Reply:
That file is called [pre-delete-cleanup_test.yaml](https://github.com/Twingate/kubernetes-operator/pull/728#diff-d29f200b6c731eba23039df74be09f98ae28e74a85ccfc77c90b8a9ffb2188a3) but it tests 2 things: the subchart and the clean up job. I think they should be separately so when we make changes to the subchart, only that test should fail.

Reply:
Updated in [493c647](https://github.com/Twingate/kubernetes-operator/pull/728/commits/493c6477b0e8e2416804667ddb5047f156a701dc)